url-gen/src/main/kotlin/com/cloudinary/transformation/resize/Fill.kt

```diff
@@ -25,30 +25,30 @@ class Fill(
 
     class Builder : BaseBuilder<Builder>() {
         private var gravity: Gravity? = null
-        private var offsetX: Any? = null
-        private var offsetY: Any? = null
+        private var x: Any? = null
```

Comment:
It's worth adding a comment that x,y are only working with `g_xy_center`.

The test should also reflect that

Reply:
@patrick-tolosa , x and y are also supported for fill and other crop modes as well